tests/common/dualtor/dual_tor_io.py

```diff
@@ -327,8 +330,8 @@ def send_and_sniff(self, sender, sniffer):
         """
         @summary: This method starts and joins two background threads in parallel: sender and sniffer
         """
-        self.sender_thr = threading.Thread(target=sender)
-        self.sniff_thr = threading.Thread(target=sniffer)
+        self.sender_thr = InterruptableThread(target=sender)
```

Comment:
Do we need to call `set_error_handler` for these two?

Reply:
Yes, looks like we should put a `set_error_handler` for the sniffer thread so set the `sniffer_started` event to free the sender thread, will update.

Reply:
Does the sender thread also need an error handler? `run_test` in `data_plane_utils.py` waits for the sender to set `io_ready_event` so any errors here might cause a the main thread to hang.

Reply:
Updated, thanks!